release-notes/release-notes-2-6.md

```diff
@@ -79,24 +79,37 @@ there will be multiple metadata instances that represent the same real-world "th
 [0465 Duplicate Processing](../open-metadata-publication/website/open-metadata-types/0465-Duplicate-Processing.md)
 types allow these elements to be linked together.
 
-## Presentation Server / React UI
+### Presentation Server / React UI
 
-The node based User Interface component known as 'Presentation Server' has now fully moved to it's own
+* The node based User Interface component known as 'Presentation Server' has now fully moved to it's own
 [GitHub Repository](https://github.com/odpi/egeria-react-ui). 
-The docker image has been renamed to [egeria-react-ui](https://hub.docker.com/repository/docker/odpi/egeria-react-ui)  
+* The docker image has been renamed to [egeria-react-ui](https://hub.docker.com/repository/docker/odpi/egeria-react-ui) 
+* Dino - Adds display of integration servers’ integration services and engine hosts’ engine services, including display of a dependency on a partnerOMAS.
+* Rex - Improved error reporting and geometry management plus more consistent handling of focus objects.
+Enterprise queries are now the default, but can be over-ridden to perform a local operation.
+* At this time 'Server Author' and 'Glossary Author' are still in development.
 
-Bug fixes & Improvements have been made in Dino, Type Explorer & Repository Explorer.
-At this time 'Server Author' and 'Glossary Author' are still in development.
-
-## New Helm Chart
+### New Helm Chart
 
 In addition to our 'lab' helm chart to support the Coco Pharmaceuticals environment, we have now added an [additional
 helm chart](../open-metadata-resources/open-metadata-deployment/egeria-base) which provides a simpler environment with just a single platform, and a single server, but configured with 
 persistence and auto start. This offers an example of a simple Kubernetes deployment.
 
-## Other changes
+### Graph Repository
+* Now implements the findEntities and fnidRelationships methods of the OMRS MetadataCollection API.
+Added detailed documentation for the graph repository
+
+### Conformance Test Suite
+* Now implements the findEntities and fnidRelationships methods of the OMRS MetadataCollection API.
```

Comment:
This doubles up on the graph repo comments - I thinker the CTS we just want to say something like:
"CTS now has tests for findEntities and findRelationships methods and search tests have been realigned into profiles so that all search operations are in optional profiles, with basic and advanced profiles for each of entities and relationships."

Reply:
fixed